ejb3/src/main/java/org/jboss/as/ejb3/subsystem/EJB3RemoteResourceDefinition.java

```diff
@@ -77,6 +77,13 @@
                     .setFlags(AttributeAccess.Flag.RESTART_ALL_SERVICES)
                     .build();
 
+    static final SimpleAttributeDefinition DEFAULT_RESPONSE_COMPRESSION =
```

Comment:
Are the management model changes complete?  I can see this being used in the add operation but I don't see it on the resource and the other related changes for a model update.

Reply:
This was a bug - there are no subsystem changes.